server/apps/comments/__init__.py

```diff
@@ -11,9 +11,13 @@
 """Generic comments module."""
 from .comments import CommentsService, CommentsResource, comments_schema  # noqa
 import superdesk
+from .user_mentions import on_activity_updated
 
 
 def init_app(app):
     endpoint_name = 'comments'
     service = CommentsService(endpoint_name, backend=superdesk.get_backend())
     CommentsResource(endpoint_name, app=app, service=service)
+
+    app.on_updated_activity -= on_activity_updated
```

Comment:
These hooks are already created by the Eve framework. As an general convention, should be avoid this? 
Moreover, I created an event hook on updating roles when privileges are revoke. If we don't create event hook the core will he calling the `Desks` service api. Is this acceptable?